e2e/melange/src/ui/ui.re

```diff
@@ -1,10 +1,22 @@
 [%styled.global {|
+/* comment */
+/** comment
+ multiline 
+*/
   div {
-    background-color: green;
+/** comment
+ multiline 
+*/
+    background-color: green; /* another one */
   }
 
   @media (min-width: 400px) {
+    /* comment 1 */
     div {
+      padding: 10px;
+/** comment
+ multiline 
+*/
```

Comment:
Would you mind adding this as a cram test?

Reply:
sure!